idaes/generic_models/properties/core/eos/tests/test_enrtl_verification_NaCl_KCl.py

```diff
@@ -873,36 +904,511 @@ def test_log_gamma_salt(self, model):
+                0.217230316409124: -0.10177066336412,
+                0.242098601913171: -0.101795262910288,
+                0.257473352254809: -0.101423890651408,
+                0.277248565121413: -0.101168550217065,
+                0.299149374540103: -0.101158577170881,
+                0.322913066330285: -0.101106699929557,
+                0.345124356144224: -0.10106579358282,
+                0.367335645958163: -0.101004359292059,
+                0.499767476085357: -0.10135461223899,
+                0.56747608535688: -0.100892692714428,
+                0.589687375170819: -0.100985218003855,
+                0.613451066961001: -0.101015452538631,
+                0.63685651214128: -0.101004794321336,
+                0.66026195732156: -0.100994136104041,
+                0.68366740250184: -0.101270869103099,
+                0.707072847682119: -0.101547602102156,
+                0.72964238410596: -0.101537324535479,
+                0.751376011773363: -0.101919324732628,
+                0.768762913907285: -0.101519510086571,
+                0.788155997056659: -0.102085461424947,
+                0.811561442236939: -0.102074803207652,
+                0.834966887417219: -0.102638927423062,
+                0.858372332597498: -0.102628269205767,
+                0.881777777777778: -0.103228317323222,
+                0.905183222958057: -0.103397278616147,
+                0.928588668138337: -0.103674011615205,
+                0.970384105960265: -0.103439435672056}
+
+        for x, g in data.items():
+            m = 4  # molality of solution
+            w = 1000/18  # mol H2O per kg H2O
+            k = x*m  # mol K+
+            n = m-k  # mol Na+
+
+            # Set composition
+            model.state[1].mole_frac_phase_comp["Liq", "H2O"].set_value(
+                w/(w+2*m))
+            model.state[1].mole_frac_phase_comp["Liq", "K+"].set_value(
+                k/(w+2*m))
+            model.state[1].mole_frac_phase_comp["Liq", "Na+"].set_value(
+                n/(w+2*m))
+            model.state[1].mole_frac_phase_comp["Liq", "Cl-"].set_value(
+                m/(w+2*m))
+
+            assert pytest.approx(0.06293706294, rel=1e-8) == value(
+                model.state[1].mole_frac_phase_comp["Liq", "Cl-"])
+            assert pytest.approx(0.874125874, rel=1e-8) == value(
+                model.state[1].mole_frac_phase_comp["Liq", "H2O"])
+            assert value(model.state[1].Liq_ionic_strength) == pytest.approx(
+                0.06293706294, rel=1e-8)
+
+            # Calculate mean activity coefficient for NaCl
+            # Natural log, mole fraction basis
+            ln_pm = value(0.5*(model.state[1].Liq_log_gamma["Na+"] +
+                               model.state[1].Liq_log_gamma["Cl-"]))
+
+            # Convert to log base 10 and molal basis
+            # Eqn 28 [1] - note that m=4 appears to be based on total salt
+            log_m = ln_pm/log(10) - log(1+18*2*4/1000, 10)
```

Comment:
Just for readability purposes, I think it would be more clear to write `log((w+2m)/w, 10)` instead of `log(1+18*2*4/1000, 10)`.